QATCH/processors/Analyze.py

```
@@ -3970,22 +3971,23 @@ def _restore_qmodel_predictions(self):
-                    from traceback import format_tb
 
                     a_list = ["Traceback (most recent call last):"]
                     a_list = a_list + format_tb(tb, limit)
                     a_list.append(f"{t.__name__}: {str(v)}")
                     for line in a_list:
                         Log.d(line)
+
                     Log.e(e)
                     Log.e(
                         TAG,
                         f"Error using 'QModel Indus'... Using a fallback model for auto-fitting.",
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)